dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -87,9 +87,23 @@ async def get_actor_info(self):
                 "current_worker_id": actor_table_entry.address.worker_id.hex(),
                 "current_raylet_id": actor_table_entry.address.raylet_id.hex(),
                 "ip_address": actor_table_entry.address.ip_address,
-                "port": actor_table_entry.address.port
+                "port": actor_table_entry.address.port,
+                "metadata": dict()
             }
             actors[actor_id] = entry
+
+            deployments = await self.get_serve_info()
```

Comment:
on exception, it returns "{}", and it's not a dict. so the next will fail 
AttributeError: 'str' object has no attribute 'items'

Reply:
Oh thanks for catching that!